src/pybritive/completers/__init__.py

```diff
@@ -1,11 +1,2 @@
 # register the completion scripts since they are not required to be registered anywhere else
-
-import pkg_resources
-
-# click < 8.0.0 does shell completion different...
-# not all the classes/decorators are available, so we cannot
-# create custom shell completions like we can with click > 8.0.0
-click_major_version = int(pkg_resources.get_distribution('click').version.split('.')[0])
-if click_major_version >= 8:
-    from . import powershell_completion
-    from . import bash_gte_42
+from . import bash_gte_42, powershell_completion  # noqa: F401
```

Comment:
not sure we can stop supporting click 7.x, as other CLI tools (Azure CLI for sure) requires click 7.x. can we confirm that python3.8 and up are not supported by click 7.x?